docs/poisoning_object_detection.md

```diff
@@ -1,12 +1,11 @@
 # Object Detection Poisoning
 
-Object Detection Poisoning (using the YOLOv3 model) requires [PytorchYolo](https://github.com/eriklindernoren/PyTorch-YOLOv3) (with its accompanying dependencies).  Armory currently does not include this in the base docker image, neither does it maintain a separate docker image just to support this.
-
-To run, please install via ```pip install .[yolo]``` and run using ```--no-docker```.
+Object Detection Poisoning (using the YOLOv3 model) requires [PytorchYolo](https://github.com/eriklindernoren/PyTorch-YOLOv3) (with its accompanying dependencies).  Armory currently does not include this in the base docker image, however, support does exist in the `twosixarmory/yolo` image.
```

Comment:
This one can be removed now

Reply:
Updated to remove redundant documentation.